packages/xml-views-validation/src/utils/custom-class.ts

```diff
@@ -0,0 +1,36 @@
+import { SVG_NS, TEMPLATING_NS, XHTML_NS } from "./special-namespaces";
```

Comment:
probably copy->pasta, but please move the import of `"./special-namespaces";` to be 
the **last** import in this file.